admin_ui/package.json

```diff
@@ -18,13 +18,16 @@
         "@fortawesome/vue-fontawesome": "^0.1.10",
         "@types/js-cookie": "^2.2.6",
         "axios": "^0.21.2",
+        "chart.js": "^4.3.0",
         "core-js": "^3.6.5",
         "flatpickr": "^4.6.9",
+        "highcharts": "^11.0.1",
         "is-svg": "^4.3.1",
         "js-cookie": "^2.2.1",
         "json-bigint": "^1.0.0",
         "ssri": "^8.0.1",
         "vue": "^2.7.14",
+        "vue-chartkick": "^0.6.1",
```

Comment:
What's the interaction between `chart.js`, `highcharts` and `vue-chartkick`?

Reply:
Is `chart.js` redundant? It looks like we're just using `highcharts`?

Reply:
Have you tried `chart.js`? It's slightly smaller then highcharts, and it supports tree shaking, so we might get a smaller bundle size:

https://bundlephobia.com/package/chart.js@4.3.0
https://bundlephobia.com/package/highcharts@11.1.0

Also, the licensing on highcharts looks complicated, so I'd rather stay away from it. chart.js is MIT like Piccolo Admin.

https://github.com/chartjs/Chart.js/blob/master/LICENSE.md

Reply:
My mistake. I tried both libraries, so it accidentally stayed in `package.json`. This PR uses `highcharts`, but I will switch to `Chart.js` in the next commit.